recipes/imapclient/meta.yaml

```diff
@@ -0,0 +1,69 @@
+{%set name = "IMAPClient" %}
+{%set version = "1.0.2" %}
+{%set compress_type = "tar.gz" %}
```

Comment:
I think this is too much. I have a script that I want to send to conda-forge soon that help maintain PyPI packages.  It will be hard to use it with all this jinja variables because the script cannot make sense of the fields below.